ckan/logic/action/create.py

```diff
@@ -1085,14 +1076,10 @@ def user_invite(context: Context,
         'role': data['role']
     }
 
-    if group.is_organization:
-        _get_action('organization_member_create')(context, member_dict)
-        group_dict = _get_action('organization_show')(context,
-                                                      {'id': data['group_id']})
-    else:
-        _get_action('group_member_create')(context, member_dict)
-        group_dict = _get_action('group_show')(context,
-                                               {'id': data['group_id']})
+    _get_action(f'{group.type}_member_create')(context, member_dict)
```

Comment:
`group.type` and `group.is_organization` aren't the same thing. You can have multiple group types and organization types.

Reply:
I wasn't aware of this, thanks